src/sage/coding/delsarte_bounds.py

```diff
@@ -220,7 +219,6 @@ def _delsarte_LP_building(n, d, d_star, q, isinteger, solver, maxc=0):
     for j in range(1, n + 1):
         rhs = sum([krawtchouk(n, q, j, r, check=False) * A[r]
```

Comment:
is't it better / faster to use `p.sum(....)` ?

Reply:
what is `p` ?

Reply:
We have above `p = MixedIntegerLinearProgram(maximization=True, solver=solver)`.

Reply:
oh, right, I overlooked this kind of syntax.